cypress/e2e/17_create_org_user.cy.ts

```diff
@@ -0,0 +1,45 @@
+describe('Add User to Organization', () => {
+  it('should add a user to an organization', () => {
+    const org = {
+      loggedInAs: 'alice',
+      name: 'Budget Org 2',
+      description: 'We are testing out our organization',
+      website: 'https://community.sphinx.chat',
+      github: 'https://github.com/stakwork/sphinx-tribes-frontend'
+    };
+
+    cy.login(org.loggedInAs);
+    cy.wait(1000);
+
+    cy.create_org(org);
+    cy.wait(1000);
+
+    cy.contains(org.name).contains('Manage').click();
+    cy.wait(1000);
+
+    cy.contains('Add User').click();
+    cy.wait(1000);
+
+    cy.get('input').type('bob');
+    cy.wait(1000);
+
+    cy.contains('Select').click();
+    cy.wait(1000);
+
+    cy.get('#sphinx-top-level-overlay').should('be.visible');
+
+    cy.get('#sphinx-top-level-overlay').within(() => {
+      cy.contains('Add User').click();
+      cy.wait(1000);
+    });
+    cy.wait(1000);
+
+    cy.get('#sphinx-top-level-overlay').within(() => {
+      cy.contains('Assign').click();
+      cy.wait(1000);
+    });
+
```

Comment:
@Vayras please click on the settings icon and confirm that the roles you assigned persisted, thanks.

Reply:
Sure will do